cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/utils/SerialisationUtil.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest.utils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SerialisationUtil {
+
+  private SerialisationUtil() {}
+
+  public static Map<String, String> stringToMap(String serialisedMap) {
+    return stringToMap(serialisedMap, ",", "=");
+  }
+
+  public static Map<String, String> stringToMap(String serialisedMap, String entrySeparator,
+      String keyValueSeparator) {
+    if (serialisedMap == null || serialisedMap.trim().isEmpty()) {
+      return new LinkedHashMap<>();
```

Comment:
is order relevant here? can we just return a regular hashmap?
Line 23 returns a standard hashmap.

Reply:
Order isn't relevant. changed to HashMap